src/main/java/org/opendcs/logging/IlexToSlf4jBridge.java

```diff
@@ -0,0 +1,46 @@
+package org.opendcs.logging;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.event.Level;
+
+public class IlexToSlf4jBridge extends ilex.util.Logger
+{
+    private Logger realLogger = null;
+
+    public IlexToSlf4jBridge(String procName)
+    {
+        super(procName, E_DEBUG3); // The bridge will capture everything.
+        realLogger = LoggerFactory.getLogger(procName);
+    }
+
+    @Override
+    public void close()
+    {
+        // nothing to do.
+    }
+
+    @Override
+    public void doLog(int priority, String text)
+    {
+        Level level = ilexLevelToSLF4J(priority);
+        realLogger.makeLoggingEventBuilder(level)
+                  .setMessage(text)
+                  .log();
+    }
+
+    private Level ilexLevelToSLF4J(int priority)
```

Comment:
Consider making ilexLevelToSLF4J a static method.

Reply:
Consider replacing with a call to JULHelpers.ilexLevelToSLF4J.  However the levels are a little different,  For example ilex.util.Logger.E_Debug3 returns Trace here, but Finest in JULHelpers.

Reply:
I'll have to review that but there are some slight differences. Though it might just be because of changes that were made as I went.